courses/serializers.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from rest_framework import serializers
+import six
+
+from .models import Course, Module
+
+
+class JsonListField(serializers.Field):
```

Comment:
If you submit some invalid JSON, a validation error is thrown but the data is persisted anyway. See this test:

    def test_bad_json(self):
        course_dict = {
            "uuid": "39b7ed5e8ace422296101a959fc3359a",
            "title": "title1",
            "author_name": "author1",
            "overview": "overview1",
            "description": "description1",
            "video_url": "http://youtube.com/1",
            "edx_instance": "edx1",
            "price_per_seat_cents": 1,
            "url": "/api/v1/coursexs/39b7ed5e8ace422296101a959fc3359a/"
        }
        resp = self.client.post('/api/v1/coursexs/', course_dict)
        self.assertEqual(resp.status_code, 201)
        course_url = json.loads(resp.content.decode('utf-8'))['url']

        module_dict = {
            "title": "title",
            "subchapters": "['a', 3]",
            "price_per_seat_cents": 3
        }
        modules_url = "{url}modules/".format(url=course_url)
        resp = self.client.post(modules_url, module_dict)
        self.assertEqual(resp.status_code, 400)  # Makes sense

        resp = self.client.get(modules_url)
        self.assertEqual(resp.status_code, 200)  # AssertionError